doc/Possible-key-values.md

```diff
@@ -32,24 +32,29 @@ Key values can be any of the following:
   + `CA:ctrl,a,ctrl,c` A key with legend CA that sends the sequence `ctrl+a`, `ctrl+c`.
   + `Cd:ctrl,backspace` A key with legend Cd that sends the shortcut `ctrl+backspace`.
 
-## Escape codes
-Value | Escape code for
+### Escape codes
+
+When defining a key value, several characters have special effects. If you want a character not to have its usual effect but to be taken literally, you should "escape" it by typing it in a special way:
+
+To get this character... | ...you can type
 :---- | :------
-`\?`  | `?`
-`\#`  | `#`
-`\@`  | `@`
-`\n`  | Literal newline character. This is different from `enter` and `action` in certain apps.
-`\t`  | Literal tab character. This is different from `tab` in certain apps.
-`\\`  | `\`
+`?` | `\?`
+`#` | `\#`
+`@` | `\@`
```

Comment:
These make sense when writing layout files for inclusion in the app. For example, `@` don't need to be escaped in custom layouts, extra keys settings or when it's not the first character of the key in the app's xml files.
They are separate from the others, perhaps this could be moved into a new paragraph and not a table as it's less important.